tests/test_wave_01.py

```diff
@@ -145,8 +146,8 @@ def test_moves_movie_from_watchlist_to_watched():
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
     # *******************************************************************************************
+    assert movie_to_watch in updated_data["watched"]
```

Comment:
Love this, simple and flexible in case we don't know for sure the order of movies in `updated_data["watched"]`